README.md

```diff
@@ -33,8 +33,7 @@ you to manage your UniFi network via the web browser. The add-on provides a
 single-click installation and run solution for Home Assistant, allowing users
 to get their network up, running, and updated, easily.
 
-This add-on supports all Home Assistant supported architectures, including the
-Raspberry Pi.
+Since version 1.0.0 of this addon, only 64-bit architectures are supported.
```

Comment:
This is not a historical document, let's drop the whole line.

Reply:
Sounds reasonable. The release notes are probably a better history.